v2/src/modules/tokens/components/TokenDetailsTransfer/TokenTransferTable.vue

```diff
@@ -7,14 +7,14 @@
                 <v-col sm="3" lg="2" class="d-none d-sm-block"> From </v-col>
                 <v-spacer v-if="lgAndUp" />
                 <v-col sm="3" lg="2" class="d-none d-sm-block"> To </v-col>
-                <v-col sm="3" :md="isNFT ? 1 : 2" :lg="isNFT ? 1 : 3">
+                <v-col :sm="isNFT ? '1' : '3'" :md="isNFT ? 1 : 2" :lg="isNFT ? 1 : 3">
```

Comment:
Not sure if this is supposed to be string or int but md and lg are receiving int

Reply:
v-col can take a `string | number | boolean`, so its not a big deal. But i will add to change it eventually ( i am pretty sure there are other places that either use string or number.